pkg/agent/interfacestore/interface_cache.go

```diff
@@ -42,6 +42,10 @@ const (
 	interfaceIPIndex = "ip"
 	// ofPortIndex is the index built with InterfaceConfig.OFPort
 	ofPortIndex = "ofPort"
+	// externalEntityIndex is the index built with InterfaceConfig.EntityNamespace + EntityName.
+	// Only the interfaces of an ExternalEntity get indexed.
+	// One ExternalEntity may get more than one interface.
```

Comment:
Why one ExternalEntity may get more than one interface? I think we will generate an ExternalEntity for each interface.

Reply:
Will remove this comment line